api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -721,6 +726,10 @@ private String getQualifiedColumnName(Domain currentDomain, String columnName) {
     final String cohortSampleNamesFilename = "cohort_sample_names_" + qualifier + ".txt";
     final String cohortSampleMapFilename = "cohort_sample_map_" + qualifier + ".csv";
     final String cohortVcfFilename = "cohort_" + qualifier + ".vcf";
+    // TODO(RW-5735): Writing to the "tmp" dataset is a temporary workaround.
+    final String cohortExtractTable =
+        "fc-aou-cdr-synth-test.tmp_shared_cohort_extract."
```

Comment:
presumably we will eventually make this a per env parameter?

Reply:
Per above comment, this line in particular is temporary. There's another ticket linked in the other TODO's here to plumb through env vars for read access to the CDR dataset as well.

Reply:
gotcha, thank you